LoginPlugin/iOS/ZappLoginPluginExample-iOS/SelectionScreenViewController.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SelectionScreenViewController.swift
+//  ZappLoginPluginExample-iOS
+//
+//  Created by Roi Kedarya on 17/05/2018.
+//  Copyright © 2018 Applicaster. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import ApplicasterSDK
+
+@objc class SelectionScreenViewController: UIViewController {
```

Comment:
can you add a comment in a few words what this class is doing?

Reply:
done